component/service/src/main/java/org/exoplatform/social/rest/api/RestUtils.java

```diff
@@ -345,4 +358,60 @@ public static IdentityManager getIdentityManager() {
     }
     return identityManager;
   }
+
+  public static String getBaseURLSiteRest() {
+    return "/" + PortalContainer.getCurrentPortalContainerName() + "/" + PortalContainer.getCurrentRestContextName()
+        + BASE_URL_SITE_REST_API;
+  }
+
+  public static String buildAttachmentUrl(String siteName, String type) {
```

Comment:
why not adding that here https://github.com/Meeds-io/social/blob/84be6394c4a559220ef01c6b26e421b73ff0acac/component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java#L47 ?